client/tests/integration/permissions/mod.rs

```diff
@@ -210,23 +194,14 @@ fn account_can_query_only_its_own_domain() {
 // If permissions are checked after instruction is executed during validation this introduces
 // a potential security liability that gives an attacker a backdoor for gaining root access
 fn permissions_checked_before_transaction_execution() {
-    let instruction_judge = JudgeBuilder::with_validator(
-        private_blockchain::register::GrantedAllowedRegisterDomains.into_validator(),
-    )
-    .at_least_one_allow()
-    .build();
-
-    let (_rt, _not_drop, iroha_client) = <PeerBuilder>::new()
-        .with_instruction_judge(Box::new(instruction_judge))
-        .with_query_judge(Box::new(DenyAll::new()))
-        .start_with_runtime();
+    let (_rt, _not_drop, iroha_client) = <PeerBuilder>::new().start_with_runtime();
 
     let isi = [
         // Grant instruction is not allowed
-        Instruction::Grant(GrantBox::new(
-            PermissionToken::from(private_blockchain::register::CanRegisterDomains::new()),
-            IdBox::AccountId("alice@wonderland".parse().expect("Valid")),
-        )),
+        // Instruction::Grant(GrantBox::new(
+        //     PermissionToken::from(private_blockchain::register::CanRegisterDomains::new()),
+        //     IdBox::AccountId("alice@wonderland".parse().expect("Valid")),
+        // )),
```

Comment:
commented code?

Reply:
Indeed, thanks!

Howerver I looked at this tests and it seems to check nothing. I decided to remove it, because we already check similar thing in another tests